ui/src/app/layout/dashboard/dashboard.component.ts

```diff
@@ -43,14 +43,20 @@ export class DashboardComponent implements OnInit {
 
     configTemplatesCopiedValue = false;
 
+    configAmountOfEntries: Observable<number>;
+
     constructor(private environments: EnvironmentsService, private applicationsService: ApplicationsService,
                 private serverInfoService: ServerInfoService, private location: Location,
                 private translate: TranslateService) {
         this.allEnvironments = environments.getEnvironments();
         this.selectedEnvironment = environments.getCurrentEnvironment();
         this.serverInfos = environments.getCurrentEnvironmentServerInfo();
-        this.changelog = this.selectedEnvironment.pipe(flatMap(env => environments.getChangeLog(env.id)))
-            .pipe(map(changes => this.formatChanges(changes))).pipe(shareReplay(1));
+        this.configAmountOfEntries = this.serverInfoService.getUiConfig().pipe(map(config => config.changelogEntries));
+
+        this.changelog = this.serverInfoService.getUiConfig().pipe(switchMap(config =>
+            this.selectedEnvironment
+                .pipe(flatMap(env => this.environments.getChangeLog(env.id)))
```

Comment:
Isn't `getChangeLog()` hard-limiting the results of the REST API call to 100? (Query param to endpoint)

Should perhaps be raised to 1,000.